README.md

```diff
@@ -126,13 +126,17 @@ We are now using MongoDB in place of Trello. In order to configure this applicat
 DBNAME=[Your DB Name Here]
 CLIENT=[Your Connection String Here] 
 
-2. Add CLIENT secret to Github
+2. Add CLIENT secrets to Github
 3. Add CLIENT and some new DB eg 'PRODDB' as DBNAME as Heroku config variables
 ```
 
 ## Linking to OAuth
 To-Do App is using GitHub for linking to default OAuth provider to manage user access. Users will now be redirected to github to sign and access the application. You will need to register your application with OAuth provider and populate your .env file with the enviroment variables found in the .env.template with the values taken from Github OAuth. Users with "read" access will not be allowed to add, update or delete the tasks in the app.
 
+## Send logs to Loggly
+- sudo pip install loggly-python-handler
```

Comment:
You shouldn't need to install this with pip if you've followed the `poetry add loggly-python-handler` step below, since then poetry will be managing that dependency (as long as you're running the app with poetry)